src/scrapers/leumi.ts

```diff
@@ -25,18 +25,18 @@ const INVALID_PASSWORD_MSG = 'אחד או יותר מפרטי ההזדהות ש
 function getPossibleLoginResults() {
   const urls: LoginOptions['possibleResults'] = {
     [LoginResults.Success]: [/ebanking\/SO\/SPA.aspx/i],
-    [LoginResults.InvalidPassword]: [async options => {
-      if (!options || !options.page) {
-        throw new Error('missing page options argument');
-      }
-
-      const errorMessage = await (0, _elementsInteractions.pageEvalAll)(options.page, '.errHeader', [], label => {
-        var _ref;
-
-        return (_ref = label[0]) === null || _ref === void 0 ? void 0 : _ref.innerText;
-      });
-      return errorMessage === null || errorMessage === void 0 ? void 0 : errorMessage.startsWith(INVALID_PASSWORD_MSG);
-    }],
+    [LoginResults.InvalidPassword]: [
+      async (options) => {
+        if (!options || !options.page) {
+          throw new Error('missing page options argument');
+        }
+        const errorMessage = await pageEvalAll(options.page, '.errHeader', [], (label) => {
```

Comment:
the default value should be of the same type of the expected value, so I believe it should be `''` instead of `[]`?

Reply:
Sounds about right.
Also if that's the case, the 'LoginResults.AccountBlocked' case should be changed as well.
I'll edit a fix.